### ISSUE TITLE
STACK works with LINE but not with AREA

### ISSUE THREAD

**Issue by clement-parisot** [manuelluis/jsrrdgraph#20]
Hello,

I try to use this library to draw several stacked Areas. 
I use your test 1: https://manuelluis.github.io/jsrrdgraph/test/test1.html
![not-stack](https://cloud.githubusercontent.com/assets/6361556/5660578/1a706ff2-9722-11e4-91bb-4df4f242ec0d.png)
It seems to work perfectly with lines as you can see here:
![stack-lines](https://cloud.githubusercontent.com/assets/6361556/5660464/dbbea04a-9720-11e4-8f90-06c28c4b4832.png)
But areas seems to be supperposed:
![stack-area](https://cloud.githubusercontent.com/assets/6361556/5660483/0b9c470e-9721-11e4-90ad-72298211a1e8.png)

This is the code I use to test the feature (just add STACK and replace LINE by AREA):
```
-s end-1h
-e 1305798665
-w 600
-h 120
-v "System load"
-t Load
-E
DEF:s_avg=data/load.rrd:shortterm:AVERAGE
DEF:s_min=data/load.rrd:shortterm:MIN
DEF:s_max=data/load.rrd:shortterm:MAX
DEF:m_avg=data/load.rrd:midterm:AVERAGE
DEF:m_min=data/load.rrd:midterm:MIN
DEF:m_max=data/load.rrd:midterm:MAX
DEF:l_avg=data/load.rrd:longterm:AVERAGE
DEF:l_min=data/load.rrd:longterm:MIN
DEF:l_max=data/load.rrd:longterm:MAX
AREA:s_max#B7EFB7
AREA:s_min#FFFFFF
"AREA:s_avg#00E000: 1m average"
"GPRINT:s_min:MIN:%4.2lf Min,"
"GPRINT:s_avg:AVERAGE:%4.2lf Avg,"
"GPRINT:s_max:MAX:%4.2lf Max,"
"GPRINT:s_avg:LAST:%4.2lf Last\n"
"AREA:m_avg#0000FF: 5m average:STACK"
"GPRINT:m_min:MIN:%4.2lf Min,"
"GPRINT:m_avg:AVERAGE:%4.2lf Avg,"
"GPRINT:m_max:MAX:%4.2lf Max,"
"GPRINT:m_avg:LAST:%4.2lf Last\n"
"AREA:l_avg#FF0000:15m average:STACK"
"GPRINT:l_min:MIN:%4.2lf Min,"
"GPRINT:l_avg:AVERAGE:%4.2lf Avg,"
"GPRINT:l_max:MAX:%4.2lf Max,"
"GPRINT:l_avg:LAST:%4.2lf Last\n"
```
on this page: https://manuelluis.github.io/jsrrdgraph/test/test1.html

**Comment by manuelluis** [comment]
Fixed in 1c43e45507935fbcc99f735b11b3be97da4da91f

**Comment by clement-parisot** [comment]
Thanks you ! It works perfectly :)
